lib/spack/spack/binary_distribution.py

```diff
@@ -235,35 +250,50 @@ def sign_tarball(key, force, specfile_path):
     Gpg.sign(key, specfile_path, '%s.asc' % specfile_path)
 
 
-def generate_index(outdir, indexfile_path):
-    f = open(indexfile_path, 'w')
+def generate_html_index(path_list, output_path):
```

Comment:
Looks like this is primarily a helper function for `generate_package_index`. In that case could you indicate that and prefix it with `_`, like `_generate_html_index`?